examples/MomentoLoadGen/README.md

```diff
@@ -40,3 +40,11 @@ Within the `MomentoLoadGen` directory you can run:
 # Run example load generator
 MOMENTO_AUTH_TOKEN=<YOUR AUTH TOKEN> dotnet run
 ```
+
+If you make modifications to the code, remember to do a clean otherwise
+the program might not run.
+
+```bash
+dotnet clean
```

Comment:
FWIW I've never had to do this. but it's probably harmless?

Reply:
yep harmless

Reply:
one thing that always throws me off when working on .NET is that Visual Studio doesn't auto-save the file so if you re-run without explicitly saving then your changes won't get picked up. Any chance that that is what happened to you?